packages/orm/src/BaseEntity.ts

```diff
@@ -67,8 +68,12 @@ export abstract class BaseEntity implements Entity {
     return Object.keys(this.__orm.originalData).length > 0;
   }
 
+  get isTouchedEntity(): boolean {
```

Comment:
I think I would expect `Entity.isTouchedEntity` to become true after calling any setter.

Can we make this more hidden, not part of the public API?

Then I would care less about "what touched as a term might intuitively mean" vs. "in the joist context it specifically meant you called `em.touch`"

Reply:
hrm... .that's interesting, because you're saying "setting but not changing a value" should be considered touched? which is exactly what we don't want in this case.  If that's true, maybe we call this forceFlush or something instead of touch